modules/gdscript/gdscript_parser.cpp

```diff
@@ -2259,6 +2269,8 @@ void GDScriptParser::_parse_pattern_block(BlockNode *p_block, Vector<PatternBran
 			catch_all = catch_all || pt == PatternNode::PT_WILDCARD;
 		}
 
+		catch_all_appeared |= catch_all;
```

Comment:
(non-blocking) For consistency, this should probably be rewritten as the statement 3 lines above:
```cpp
catch_all_appeared = catch_all_appeared || catch_all;
```

Reply:
Oh sure, updated.